lab-dan/readme.md

```diff
@@ -0,0 +1,16 @@
+#Data Structures & Algorithms
+### Stacks & Queues
+
+This implementation of stack and queues are contained in separate modules, along with a shared node module (since both need simple nodes that use the same parameters--```value``` and ```previousNode```).
```

Comment:
Nice docs! Makes digging into your code quite a bit easier when there's a human readable format as a base layer to the code.